src/foam/comics/SearchMode.js

```diff
@@ -26,7 +26,11 @@ foam.ENUM({
     {
       name: 'SIMPLE',
       label: 'Simple',
-      documentation: 'A single search field that does a keyword search.'
+      documentation: 'A single search field that does a MQL and keyword search.'
```

Comment:
This is what full is supposed to be anyway. proposing:
- FULL: Both MQL and keyword
- SIMPLE: Only Keyword
- MQL: Only MQL

The validity of Simple existing at all can be debated.